contrib/cloud.google.com/go/pubsub.v1/example_test.go

```diff
@@ -0,0 +1,43 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pubsub_test
+
+import (
+	"context"
+	"log"
+
+	ddpubsub "gopkg.in/DataDog/dd-trace-go.v1/contrib/cloud.google.com/go/pubsub.v1"
```

Comment:
```suggestion
	pubsubtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/cloud.google.com/go/pubsub.v1"
```
We usually call it that in examples, adding the "trace" suffix.